apollo/lib/security/__init__.py

```diff
@@ -57,10 +57,9 @@ def get_principals(enhanced_http_connection, session):
 
         authenticated_user = enhanced_http_connection.current_user
         if authenticated_user:
-            principals += [Human, f"user:{authenticated_user.id}"]
-
             if authenticated_user.has_changed_initial_password:
                 principals.append(Authenticated)
```

Comment:
```suggestion
                principals += [Authenticated, Human,
                               f"user:{authenticated_user.id}"]
```
Followed by the removal of the next line.